example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt

```diff
@@ -0,0 +1,38 @@
+package com.checkout.example.frames.ui.screen
+
+import android.content.Context
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.runtime.Composable
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.checkout.base.model.Environment
+import com.checkout.example.frames.navigation.Screen
+import com.checkout.frames.screen.paymentform.PaymentFormConfig
+import com.checkout.frames.screen.paymentform.PaymentFormScreen
+import com.checkout.frames.style.screen.PaymentFormStyle
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent { Navigator(this.applicationContext) }
+    }
+}
+
+@Composable
+fun Navigator(context: Context) {
+    val navController = rememberNavController()
+    val defaultPaymentFormConfig = PaymentFormConfig(
+        publicKey = "",
```

Comment:
You can inject the existing test key if you like which is used in existing test sample app code.